src/vs/workbench/contrib/notebook/browser/view/renderers/backLayerWebView.ts

```diff
@@ -66,6 +67,7 @@ export interface INotebookDelegateForWebview {
 	focusNotebookCell(cell: IGenericCellViewModel, focus: 'editor' | 'container' | 'output', options?: IFocusNotebookCellOptions): Promise<void>;
 	toggleNotebookCellSelection(cell: IGenericCellViewModel, selectFromPrevious: boolean): void;
 	getCellByInfo(cellInfo: ICommonCellInfo): IGenericCellViewModel;
+	getCellByHandle(cellHandle: number): IGenericCellViewModel | undefined;
```

Comment:
`getCellByHandle` is not used anymore it seems.

Reply:
I'm not sure why you're saying that? It's used here:
https://github.com/rchiodo/vscode/blob/e2816721e14e8e69affdd0334690e66bd7266e64/src/vs/workbench/contrib/interactive/browser/interactiveEditor.ts#L152

Reply:
I mean it's not used in the webview code, so we don't need to declare it on `INotebookDelegateForWebview`, which is only used in webview.

Reply:
Ah, I see now. Will remove.